commands/channel_users.go

```diff
@@ -93,21 +93,24 @@ func channelUsersRemoveCmdF(c client.Client, cmd *cobra.Command, args []string)
 		removeAllUsersFromChannel(c, channel)
 	} else {
 		for i, user := range getUsersFromUserArgs(c, args[1:]) {
-			removeUserFromChannel(c, channel, user, args[i+1])
```

Comment:
We generally continue to the operation instead of returning, can you continue to the loop and queue up the errors within a `*multierror.Error`?

Reply:
Sure! I will adjust this

Reply:
Done @isacikgoz! 
Is a way to run these tests in my machine? I ran `make test` but all tests passed...